pkg/updatestrategy/aws_asg.go

```diff
@@ -17,20 +19,20 @@ import (
 	"github.com/aws/aws-sdk-go/service/elb"
 	"github.com/aws/aws-sdk-go/service/elb/elbiface"
 	"github.com/cenkalti/backoff"
-	log "github.com/sirupsen/logrus"
 	"github.com/zalando-incubator/cluster-lifecycle-manager/api"
 )
 
 const (
-	clusterIDTagPrefix          = "kubernetes.io/cluster/"
-	resourceLifecycleOwned      = "owned"
-	kubeAutoScalerEnabledTagKey = "k8s.io/cluster-autoscaler/enabled"
-	nodePoolTag                 = "NodePool"
-	userDataAttribute           = "userData"
-	instanceTypeAttribute       = "instanceType"
-	instanceIdFilter            = "instance-id"
-	instanceHealthStatusHealthy = "Healthy"
-	ec2AutoscalingGroupTagKey   = "aws:autoscaling:groupName"
+	clusterIDTagPrefix               = "kubernetes.io/cluster/"
+	resourceLifecycleOwned           = "owned"
+	kubeAutoScalerEnabledTagKey      = "k8s.io/cluster-autoscaler/enabled"
+	nodePoolTag                      = "NodePool"
+	userDataAttribute                = "userData"
+	instanceTypeAttribute            = "instanceType"
+	instanceIdFilter                 = "instance-id"
+	instanceHealthStatusHealthy      = "Healthy"
+	ec2AutoscalingGroupTagKey        = "aws:autoscaling:groupName"
+	instanceTerminationRetryDuration = 15
```

Comment:
I meant that the full const i.e. `instanceTerminationRetryDuration = 15 * time.Minute` should be defined here. :)